packages/react-components/react-combobox/stories/src/Dropdown/index.stories.tsx

```diff
@@ -18,6 +18,7 @@ export { Disabled } from './DropdownDisabled.stories';
 export { TruncatedValue } from './DropdownTruncation.stories';
 export { ActiveOptionChange } from './DropdownActiveOptionChange.stories';
 export { ControllingOpenAndClose } from './DropdownControllingOpenAndClose.stories';
+export { DropdownFallbackPositions } from './DropdownFallbackPositions.stories';
```

Comment:
@ValentinaKozlova let's add docs there instead https://react.fluentui.dev/?path=/docs/concepts-developer-positioning-components--default as this page documents all positioning options

Reply:
fixed :)